nationalarchives/ds-wagtail
# DF-714: Accessibility for date fields

Ticket URL: https://national-archives.atlassian.net/browse/DF-714

**Please Do Not Merge, until FE is completed**

## About these changes

- Makes the individual date fields have the label with the input. - Day, Month, Year with GDS pattern.
- Added div as with [GDS](https://design-system.service.gov.uk/components/date-input/) pattern.
- FE stylings

## How to check these changes

- Navigate to Search - Records at the National Archives
- Observe Day, Month Year labels above the input fields

## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
